stubs/Pillow/PIL/PSDraw.pyi

```diff
@@ -1,16 +1,18 @@
 from typing import Any
 
+from .Image import Image
+
 class PSDraw:
     fp: Any
     def __init__(self, fp: Any | None = ...) -> None: ...
     isofont: Any
     def begin_document(self, id: Any | None = ...) -> None: ...
     def end_document(self) -> None: ...
-    def setfont(self, font, size) -> None: ...
-    def line(self, xy0, xy1) -> None: ...
-    def rectangle(self, box) -> None: ...
-    def text(self, xy, text) -> None: ...
-    def image(self, box, im, dpi: Any | None = ...) -> None: ...
+    def setfont(self, font: str, size: int) -> None: ...
+    def line(self, xy0: tuple[int, int], xy1: tuple[int, int]) -> None: ...
+    def rectangle(self, box: tuple[int, int, int, int]) -> None: ...
+    def text(self, xy: tuple[int, int], text: str) -> None: ...
+    def image(self, box: tuple[int, int, int, int], im: Image, dpi: float | None = ...) -> None: ...
```

Comment:
These functions do a mess of different operations on the parameters, some of which may work with laxer types like Sequence or Iterable, but there's no pattern. I think the intent is to be using tuple here (as the docs say), but I have no idea how prevalent other call styles are.

Reply:
Yes, that makes sense because the number of elements is crucial for each of these.